cc/coord.cc

```diff
@@ -52,7 +52,7 @@ Coord TryParseKgs(absl::string_view str) {
 }
 
 Coord TryParseSgf(absl::string_view str) {
-  if (str.empty()) {
+  if (str.empty() || str == "tt") {
```

Comment:
Cast to lowercase first?

Reply:
Uppercase letters are reserved for representing points 27-52 on larger boards, see: https://www.red-bean.com/sgf/go.html#types

Reply:
To be clear, assuming lowercase is the correct thing to do here.

Reply:
i guess the right thing to do is to compare `(str == "tt" && boardsize ==19)`